examples/cpp/GaussianRendering.cpp

```diff
@@ -0,0 +1,28 @@
+// ----------------------------------------------------------------------------
+// -                        Open3D: www.open3d.org                            -
+// ----------------------------------------------------------------------------
+// Copyright (c) 2018-2024 www.open3d.org
+// SPDX-License-Identifier: MIT
+// ----------------------------------------------------------------------------
+
+#include <cstdlib>
+
+#include "open3d/Open3D.h"
+
+using namespace open3d;
+
+double GetRandom() { return double(std::rand()) / double(RAND_MAX); }
+
+void SingleObject() {
+    std::string file = "/home/zy/ws/Open3D/pointcloud/converted_point_cloud_29999_only_table.npz";
+    std::shared_ptr<t::geometry::PointCloud> pointcloud = std::make_shared<t::geometry::PointCloud>();
```

Comment:
shared_ptr is unnecessary - we can just instantiate directly:
`t::geometry::PointCloud pointcloud;`